inst/examples/03-formats.Rmd

```diff
@@ -103,6 +103,7 @@ bookdown::gitbook:
       size: 2
     sharing:
       facebook: yes
+      github: yes
```

Comment:
The default is `no`:

```suggestion
      github: no
```

Reply:
Yes, true! I just took it from my use-case where I wanted just this button and nothing else

Reply:
Would you change it to `no`?

Reply:
Sorry. new to pull requests. Is everything ok now?